client/apps/edit/client.coffee

```diff
@@ -27,10 +28,20 @@ async = require 'async'
       EditAdmin(article: @article, channel: channel)
       $('#edit-admin')[0]
     )
-    ReactDOM.render(
-      EditContent(article: @article, channel: channel)
-      $('#edit-content')[0]
-    )
+    if sd.EDIT_2
+      if channel.get('name') is 'Artsy Editorial'
+        # TODO - replace with better conditional (or channel.isEditorial),
+        # using name allows viewing the test channels in classic for interim.
+        @article.set('layout', 'standard')
```

Comment:
This is for development happiness rather than production -- gives me an easy way to preview both layouts by viewing a test channel or the editorial channel (isEditorial returns true for the test channels). 

Yeah, we could go ahead and convert all editorial articles to standard now to avoid this.